addons/web/static/src/core/tree_editor/condition_tree.js

```diff
@@ -901,9 +900,9 @@ export function removeWithinOperators(tree) {
         const reverse = Number.isInteger(value[0]) && value[0] > 0;
         return condition(
             path,
-            "between",
+            tree.operator === "within" ? "between" : "is_not_between",
             reverse ? Object.values(expressions) : Object.values(expressions).reverse(),
-            negate
+            negate,
```

Comment:
you should activate the tooling when working in those files (addons/web/tooling/enable.sh). No comma here.